src/html/HTMLWriter.jl

```diff
@@ -1749,11 +1822,17 @@ end
 
 function domify(dctx::DCtx)
     ctx, navnode = dctx.ctx, dctx.navnode
-    return map(getpage(ctx, navnode).mdast.children) do node
-        rec = searchrecord(ctx, navnode, node)
```

Comment:
This is the only call to `searchrecord` so if it is removed, the function `searchrecord` should also be removed.